app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ContactsController < ApplicationController
+
+  def new
+    @contact = Contact.new
+    @company = Company.find(params[:company_id])
+
+  end
+
+  def create
+    @contact = Contact.new(contact_params)
```

Comment:
are we actually associating this contact with the specific company? where are we doing that?